.github/workflows/deploy-smart.yaml

```diff
@@ -85,7 +137,7 @@ jobs:
           bundle install
       - name: Execute tests
         env:
-          SERVICE_URL: ${{ secrets.SMART_SERVICE_URL}}
+          SERVICE_URL: ${{ secrets[matrix.smartServiceURLSecretName]}}${{ matrix.serviceUrlSuffix }}
```

Comment:
Does this corresponds to `SMART_SERVICE_URL` used in [this file](https://github.com/awslabs/fhir-works-on-aws-deployment/blob/99594182336a8e0cda938643058ac3925d22dc31/integration-tests/multitenancy.test.ts)?

Reply:
yes

Reply:
nvm, actually no. this are the inferno tests.

the `SMART_SERVICE_URL` env variable used in integ tests is set here:
https://github.com/awslabs/fhir-works-on-aws-deployment/blob/3cc1379d7f0dfc31b7f35b3bd2599740c790a108/.github/workflows/deploy-smart.yaml#L198

It's a bit confusing to use different env var names for inferno tests and for integ tests, but here we are just following the existing naming convention.

Reply:
Got it! Is `${{ matrix.serviceUrlSuffix }}` needed for SMART_SERVICE_URL as well?

Reply:
No, integ tests know how to add `/tenant/<tenantId>` to the base url. Those tests deal with multiple tenants.

https://github.com/awslabs/fhir-works-on-aws-deployment/blob/3cc1379d7f0dfc31b7f35b3bd2599740c790a108/integration-tests/utils.ts#L117-L135